src/main/java/com/example/gymApp/model/User.java

```diff
@@ -1,13 +1,31 @@
 package com.example.gymApp.model;
 
+import jakarta.persistence.*;
 import lombok.Data;
+
+@Entity
+@Table(name = "users")
 @Data
 public class User {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "id")
   private Long id;
+
+  @Column(name = "first_name", nullable = false)
   private String firstName;
+
+  @Column(name = "last_name",nullable = false)
   private String lastName;
+
+  @Column(name = "username",nullable = false)
   private String username;
+
+  @Column(name = "password", nullable = false)
   private String password;
+
+  @Column(name = "is_active", nullable = false)
   private boolean isActive;
 
   public User(Long id, String firstName, String lastName, String username, String password,
```

Comment:
Are you using this constructor? It is better to use setters when creating an object, that constructor with all fields